### DISCUSSION TITLE
Wordpress Nested Pages plugin improvements

### DISCUSSION THREAD

**Discussion by Pahrosh** [community/community#120810] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Anyone knows where I can find WordPress plugin developers for developing a few new plugins, as well as improving a few existing ones? Immediate need is to add a couple features to the Nested Pages plugin https://wordpress.org/plugins/wp-nested-pages/. 

Tks
Tanin

**Comment by ebndev** [comment]
Hi @Pahrosh, thank you for being a part of the GitHub Community!

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations.

Thanks for your understanding!
